vendor/brave-ios/Ledger/BATBraveLedger.h

```diff
@@ -77,14 +77,9 @@ NS_SWIFT_NAME(BraveLedger)
 @property (nonatomic, readonly, nullable) NSString *ETHAddress;
 @property (nonatomic, readonly, nullable) NSString *LTCAddress;
 
-/// ?? Unavailable until we understand whats its for
-- (void)addressesForPaymentId:(void (^)(NSDictionary<NSString *, NSString *> *))completion NS_UNAVAILABLE;
-
 @property (nonatomic, readonly) double defaultContributionAmount;
 
-/// Retrieves the users most up to date balance to determin whether or not the
-/// wallet has a sufficient balance to complete a reconcile
-- (void)hasSufficientBalanceToReconcile:(void (^)(BOOL sufficient))completion;
+@property (nonatomic, readonly) BOOL hasSufficientBalanceToReconcile;
```

Comment:
Is there a reason this was changed? If we are reverting back to non-callback version of `hasSufficientBalanceToReconcile` then we need to update the source method to match in BATBraveLedger.mm

Reply:
fixed